optimum/exporters/openvino/convert.py

```diff
@@ -635,10 +641,14 @@ def export_from_model(
 
         logger.info(f"Automatic task detection to: {task}.")
 
+    is_encoder_decoder = getattr(getattr(model, "config", {}), "is_encoder_decoder", False)
+    model_type = getattr(getattr(model, "config", {}), "model_type", "")
     stateful = stateful and (
-        ensure_export_task_support_stateful(task)
-        or ensure_model_type_support_stateful(getattr(getattr(model, "config", {}), "model_type", ""))
+        ensure_export_task_support_stateful(task) or ensure_model_type_support_stateful(model_type)
```

Comment:
did a test with t5 and looks like it wasn't exported as a stateful model coming from https://github.com/huggingface/optimum-intel/blob/fe55db5da2e20f8df704d27c795390e3c501b606/optimum/exporters/openvino/stateful.py#L200, I think we need to include t5, mt5 and whisper to `ensure_model_type_support_stateful`

Reply:
Actually I tested again and everything works fine : I was using a tiny random model ([hf-internal-testing/tiny-random-T5Model](https://huggingface.co/hf-internal-testing/tiny-random-T5Model)) without specifying the task which was automatially detected to '`feature-extraction-with-past'`, sorry for the false alert!